client/assets/stylesheets/shared/functions/_z-index.scss

```diff
@@ -172,6 +172,7 @@ $z-layers: (
 
 		".is-section-woocommerce .global-notices": 999999,
 		".webpack-build-monitor": 99999999,
+		".tooltip.popover": 100000,
```

Comment:
I think you'd put this somewhere higher on the list, they seem to be in numerical order?

Reply:
Thanks, updated!